### ISSUE TITLE
#blocksToAvoidBreaking issue with certain blocks

### ISSUE THREAD

**Issue by 1Upped** [cabaletta/baritone#868]
1.12.2 impact client version:
#blocksToAvoidBreaking doesn't work with magma_block or magma, reason why may be because the ID of the block has been changed to "magma_block" in 1.13 from "magma" in 1.12. not sure, but may not just apply to magma but to all other blocks which have had their IDs changed from 1.11.2+

![image](https://user-images.githubusercontent.com/54632925/63882078-2c477200-c9c9-11e9-9031-26b233145341.png)

**Comment by Kataroka** [comment]
Hi. I'd like to know if this issue has been fixed yet as I am experimenting something very similar. Thank you.

**Comment by ZacSharp** [comment]
The id change will certainly cause problems when switching between 1.12.2 and 1.13+ but not during normal use.
Also I could not reproduce this. I gave Baritone a path with a magma block in the way and an alternative path with soul sand and stone floor. Without modified settings I had to use a single block of soul sand to make it break the magma block, after adding magma to `blocksToAvoidBreaking` I had to use 7 blocks of soul sand and another one it had to break until it preferred breaking the magma block. That's exactly the intended behavior, avoiding to break the block, but not at any cost.
